cas-server-support-jpa-util/build.gradle

```diff
@@ -0,0 +1,10 @@
+
+description = 'Apereo CAS JPA Ticket Registry Support'
```

Comment:
A single module with a single class in it that only does util is overkill. Can we get rid of this module? or move the class to the util package?